transform/operations.go

```diff
@@ -347,12 +347,12 @@ func (c *stringToFloat64) init(args map[string]string) error {
 func (c *stringToFloat64) transform(raw interface{}) (interface{}, error) {
 	in, ok := raw.(string)
 	if !ok {
-		return nil, errors.New("stringToFloat64 only supports strings")
+		return nil, fmt.Errorf("stringToFloat64 only supports strings")
```

Comment:
No need to `fmt.Errorf` if you're not using a format (`%`), although I suggest using `%T` here to print out the type that the function did get.

Reply:
Ya i like printing out the type.

Reply:
updated